services/catalog/src/simcore_service_catalog/schemas/project.py

```diff
@@ -0,0 +1,84 @@
+# From  project-v0.0.1.json
+# from __future__ import annotations TODO: ???
+
+# TODO: why pylint error in pydantic???
+# pylint: disable=no-name-in-module
+
+import sys
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel, EmailStr, Field
+
+current_file = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
+
+KEY_RE = r"^(simcore)/(services)(/demodec)?/(comp|dynamic|frontend)(/[^\s]+)+$"
+VERSION_RE = r"^(0|[1-9]\d*)(\.(0|[1-9]\d*)){2}(-(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*)(\.(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*))*)?(\+[-\da-zA-Z]+(\.[-\da-zA-Z-]+)*)?$"
```

Comment:
is that for the version of a node? I don't think  that is compatible with the current way. also it is not in sync with node jsonschema.
I'm also wondering if the integration version should not be in a docker label instead?